posthog/models/property.py

```diff
@@ -51,6 +101,28 @@ class Property:
     type: PropertyType
     group_type_index: Optional[GroupTypeIndex]
 
+    # Type of `key`
+    event_type: Optional[Literal["events", "actions"]]
+    # Query people who did event '$pageview' 20 times in the last 30 days
+    # translates into:
+    #  operator_value = 30, operator_interval = day
+    # time_value = 30, time_interval = day ??
```

Comment:
@EDsCODE could you help me fill these comments up? Wasn't sure what you had in mind here, and wanted a way to parse these